### PR TITLE
Bug 1342495 - Allow perfherder to alert on abs change instead of percentage

### PR DESCRIPTION
**Pull request by wlach**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2242)

### CODE REVIEW

File: tests/perfalert/test_create_alerts.py [hunk 1]
```diff
@@ -166,3 +169,32 @@ def test_custom_alert_threshold(
...
+
+
+@pytest.mark.parametrize(('new_value', 'expected_num_alerts'),
+                         [(1.0, 1), (0.25, 0)])
+def test_alert_change_type_absolute(test_repository,
+                                    failure_classifications,
+                                    generic_reference_data,
+                                    test_perf_signature, new_value,
+                                    expected_num_alerts):
+    # modify the test signature to say taht we alert on absolute value
```
Comment (rwood-moz) [inline tests/perfalert/test_create_alerts.py]: nit typo

File: treeherder/etl/perf.py [hunk 2]
```diff
@@ -151,6 +153,8 @@ def _load_perf_datum(job, perf_datum):
                     # null (None). Null indicates no preference has been
                     # set.
                     'should_alert': subtest.get('shouldAlert'),
+                    'alert_change_type': PerformanceSignature._get_alert_change_type(
+                        suite.get('alertChangeType')),
```
Comment (rwood-moz) [inline treeherder/etl/perf.py]: Should this be subtest.get? Or are you using the setting in suite here intentionally to force all subtests to have the same setting as the suite they belong to? In which case is there a point in having the 'alertChangeType' setting in the performance-artifact.json?

Comment (wlach) [inline treeherder/etl/perf.py]: Yes, it should be subtest.get. Good catch!

**Comment by wlach** [comment]
Hey @rwood-moz, I think this is ready for another look. I kept the fixes in a seperate commit for easier reviewing.

**Pull request merged**